gtk4/src/auto/drop_down.rs

```diff
@@ -65,6 +65,8 @@ impl DropDown {
 
     #[doc(alias = "gtk_drop_down_get_enable_search")]
     #[doc(alias = "get_enable_search")]
+    #[doc(alias = "enable-search")]
```

Comment:
Duplicated doc aliases... looks like you have to fix gir :P

Reply:
https://github.com/gtk-rs/gir/pull/1572

Reply:
You can regen now

Reply:
Regenerated.